packit/config.py

```diff
@@ -104,6 +104,23 @@ def is_dict_valid(cls, raw_dict: dict) -> bool:
         return Draft4Validator(JOB_CONFIG_SCHEMA).is_valid(raw_dict)
 
 
+class SyncFilesConfig(Dict):
```

Comment:
Why inherit from dict? I'd say this class should have the whole list inside, not just one item.